index.html

```diff
@@ -77,6 +77,62 @@ <h1 class="primary-slider__title font-secondary">Beautiful Escape</h1>
         </div>
     </div>
 
+    <div class="section-media-content">
+        <div class="quote">
+            <p class="quote__content">Surfing is the most blissful experience you can have on this planet, a taste of heaven.</p>
+            <span class="quote__author">john mccarthy</span>
+
+            <svg class="quote__line" xmlns="http://www.w3.org/2000/svg" width="2" height="100" viewBox="0 0 2 100">
+                <path fill="#5da8ef" d="M0 100V0h2v100z"/>
+            </svg>
+
+        </div>
+
+        <div class="gallery">
+            <img class="gallery__left-image" src="images/surfer.jpg" alt="surfer image" />
+            <img class="gallery__right-image" src="images/surfer-waves.jpg" alt="surfer on waves image" />
```

Comment:
`left` and `right` are not very good naming practices in this case. Let's discuss why.